.github/workflows/get-version.yml

```diff
@@ -156,10 +161,20 @@ jobs:
               fi
               ;;
             *)
+              # Any branch name
+              GITHUB_BRANCH_BASE_REF_NAME="$(gh pr view $BRANCHNAME -q .baseRefName --json headRefName,baseRefName,state)"
+              GITHUB_BRANCH_PR_STATE="$(gh pr view $BRANCHNAME -q .state --json headRefName,baseRefName,state)"
               echo "release=`date +%s`.`echo ${{ github.sha }} | cut -c -7`" >> $GITHUB_OUTPUT
-              echo "release_cloud=$GITHUB_RELEASE_CLOUD" >> $GITHUB_OUTPUT
-              echo "release_type=$GITHUB_RELEASE_TYPE" >> $GITHUB_OUTPUT
-              ;;
+
+              # Check if pull request branch targeting develop
```

Comment:
```suggestion
              # Check if the pull request is targeting develop
```

Reply:
(should probably done in every PR if it's not too much of a hassle since it's really minor)

Reply:
will handle it separately to avoid remove all reviews